org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/core/compiler/InvalidInputException.java

```diff
@@ -28,7 +28,7 @@ public class InvalidInputException extends Exception {
  * Creates a new exception with no detail message.
  */
 public InvalidInputException() {
-	super();
+	org.eclipse.jdt.internal.compiler.parser.Scanner.invalidInput();
```

Comment:
Not sure I understood why `Scanner.invalidInput()` is added here?

Reply:
thanks. left over from refactoring